journal-of-universal-computer-science.csl

```diff
@@ -0,0 +1,530 @@
+  <info>
+    <title>Journal of Universal Computer Science</title>
+    <title-short>J.UCS</title-short>
+    <id>http://www.zotero.org/styles/journal-of-universal-computer-science</id>
+    <link href="http://www.zotero.org/styles/journal-of-universal-computer-science" rel="self"/>
+    <link rel="template" href="http://www.zotero.org/styles/apa-single-spaced"/>
+    <link href="http://www.jucs.org/ujs/jucs/info/submissions/style_guide.html" rel="documentation"/>
+    <author>
+      <name>Georg Link</name>
+      <email>glink@unomaha.edu</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="science"/>
+    <issn>0948-695x</issn>
+    <eissn>0948-6968</eissn>
+    <updated>2016-01-10T17:02:33+00:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="en">
+    <terms>
+      <term name="editortranslator" form="short">
+        <single>ed. &amp; trans.</single>
+        <multiple>eds. &amp; trans.</multiple>
+      </term>
+      <term name="translator" form="short">
+        <single>trans.</single>
+        <multiple>trans.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="container-contributors">
+    <choose>
+      <if type="chapter paper-conference entry-dictionary entry-encyclopedia" match="any">
+        <names variable="editor translator container-author" delimiter=", " suffix=", ">
+          <name and="symbol" initialize-with=". " delimiter=", "/>
+          <label form="short" prefix=" (" text-case="title" suffix=")"/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="secondary-contributors">
+    <choose>
+      <if type="article-journal chapter paper-conference entry-dictionary entry-encyclopedia" match="none">
+        <names variable="translator editor container-author" delimiter=", " prefix=" (" suffix=")">
+          <name and="symbol" initialize-with=". " delimiter=", "/>
+          <label form="short" prefix=", " text-case="title"/>
+        </names>
+      </if>
+    </choose>
+  </macro>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" sort-separator=", " initialize-with=". " delimiter=", " delimiter-precedes-last="never" delimiter-precedes-et-al="always"/>
+      <label form="short" prefix=" (" suffix=")" text-case="capitalize-first"/>
+      <substitute>
+        <names variable="editor"/>
+        <names variable="translator"/>
+        <choose>
+          <if type="report">
+            <text variable="publisher"/>
+            <text macro="title"/>
+          </if>
+          <else>
+            <text macro="title"/>
+          </else>
+        </choose>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-short">
+    <names variable="author">
+      <name form="short" and="text" initialize-with=". "/>
```

Comment:
you want to remove the `and="text"` here to remove the and between authors. That was what you were asking about, right?